generators/app/templates/package.json

```diff
@@ -26,7 +26,8 @@
   },
   "peerDependencies": {
     "react": "^15.4.2",
-    "react-dom": "^15.4.2"
+    "react-dom": "^15.4.2",
+    "terra-base": "^0.x"
```

Comment:
I think, any peer Dependencies should also be dependencies. So, something like this?

```json
  "devDependencies": {
    "terra-toolkit": "^0.x"
  },
  "peerDependencies": {
    "react": "^15.4.2",
    "react-dom": "^15.4.2",
    "terra-base": "^0.x"
  },
  "dependencies": {
    "classnames": "^2.2.5",
    "react": "^15.4.2",
    "react-dom": "^15.4.2",
    "terra-mixins": "^1.0.0",
    "terra-base": "^0.x"
  },
```